main.go

```diff
@@ -144,6 +144,17 @@ func init() {
 	codeCmd.Flags().IntVar(&config.HDD, "hdd", 0, "Amount of hard-disk space to allocate in GB")
 	rootCmd.AddCommand(codeCmd)
 
+	cpCmd := &cobra.Command{
+		Use:     "cp",
+		Short:   "Copy files and folders to or from a remote host",
+		Long:    "Example: unweave cp local/path sess:<exec-name/id>/absolute/path/on/remote",
```

Comment:
Docs need to have a format for the arguments and something the user can understand (in this case, the user does not understand the concept of `exec`:

```go
Use:   "cp <source-path> <destination-path>",
Short: "Copy files and folders to or from a remote host",
Long: wordwrap.String("Copy files and folders to or from a remote host \n\n"+
	"cp <source-path> <destination-path>\n\n"+
	"To copy to a remote session, you must prefix the remote path with the session "+
	"name `sess:<session-name>`. The `sess:` prefix is necessary.\n\n"+
	"The full syntax for this command is:\n\n"+
	"Local to remote:\n"+
	"unweave cp <local-path> sess:<session-name><remote-path>\n\n"+
	"Remote to local:\n"+
	"unweave cp sess:<session-name><remote-path> <local-path> \n\n"+
	"Example: \n"+
	"\tunweave cp /home/data sess:session-name/home/ml-data\n"+
	"\tunweave cp sess:session-name/home/ml-data /home/data\n", ui.MaxOutputLineLength),
```